src/FeynmanTheory.jl

```diff
@@ -37,33 +37,58 @@ F(v,w,α)=(3/(4*v))*(v-w)^2-AF(v,w,α)
 
 # Let's wrap the Feynman athermal variation approximation in a simple function
 """
-    feynmanvw(α; v=7.0, w=6.0)
+    feynmanvw(α; v = 0.0, w = 0.0)
 
     Calculate v and w variational polaron parameters,
     for the supplied α Frohlich coupling.
     This version uses the original athermal action (Feynman 1955).
 	Returns v,w.
 """
-function feynmanvw(α; v=7.0, w=6.0) # v,w defaults
-    initial=[v,w]
-    # Main use of these bounds is stopping v or w going negative, at which you get a NaN error as you are evaluating log(-ve Real)
-    lower=[0.1,0.1]
-    upper=[1000.0,1000.0]
+function feynmanvw(α; v = 0.0, w = 0.0) # v, w defaults
 
-    myf(x) = F(x[1],x[2],α) # Wraps the function so just the two variational params are exposed, so that Optim can call it
-
-    # Now updated to use Optim.jl > 0.15.0 call signature (Julia >0.6 only)
-    res=optimize(OnceDifferentiable(myf, initial; autodiff = :forward), 
-                 lower, upper, initial, Fminbox( BFGS() ) )
-    # specify Optim.jl optimizer. This is doing all the work.
+    # Intial guess for v and w.
+    if v == 0.0 || w == 0.0 # Default values to start with. Generates a random float between 1.0 and 11.0
```

Comment:
I don't think this is great style... I think better would be to keep this function just taking a v and w parameter, then wrap this function with a 'random restarts' or 'checked convergence' function that tries the random points & restarts if convergence is not achieved.

But we can probably sort this out when developing / standardising the multiple phonon branch version!

Reply:
That does sound better! I wonder when we constrain v and w in a box, can we also constrain w < v during the optimisation? Would avoid the nasty busy that happens when w >= v!